modules/api-resources/api-resources-full/src/main/webapp/WEB-INF/web.xml

```diff
@@ -24,108 +24,359 @@
             </param-value>
         </init-param>
     </servlet>
     <servlet-mapping>
-        <servlet-name>IdentityUserV1ApiServlet</servlet-name>
-        <url-pattern>/identity/user/v1.0/*</url-pattern>
+        <servlet-name>OrganizationServerV1Servlet</servlet-name>
+        <url-pattern>/o/server/v1/*</url-pattern>
     </servlet-mapping>
 
-    <!--  Servlet instance for /identity/recovery/v0.9 -->
+    <!--  Servlet instance for /o/server/v2 -->
     <servlet>
-        <servlet-name>IdentityRecoveryV0_9ApiServlet</servlet-name>
+        <servlet-name>OrganizationServerV2Servlet</servlet-name>
         <servlet-class>
             org.apache.cxf.jaxrs.servlet.CXFNonSpringJaxrsServlet
         </servlet-class>
         <load-on-startup>1</load-on-startup>
         <init-param>
             <param-name>jaxrs.serviceClasses</param-name>
             <param-value>
-                org.wso2.carbon.identity.recovery.endpoint.ClaimsApi,
-                org.wso2.carbon.identity.recovery.endpoint.RecoverPasswordApi,
-                org.wso2.carbon.identity.recovery.endpoint.RecoverUsernameApi,
-                org.wso2.carbon.identity.recovery.endpoint.SetPasswordApi,
-                org.wso2.carbon.identity.recovery.endpoint.ValidateCodeApi,
-                org.wso2.carbon.identity.recovery.endpoint.CaptchaApi
+                org.wso2.carbon.identity.rest.api.server.email.template.v2.EmailApi
             </param-value>
         </init-param>
         <init-param>
             <param-name>jaxrs.providers</param-name>
             <param-value>
+                org.wso2.carbon.identity.servlet.mgt.CustomJacksonJsonProvider,
                 com.fasterxml.jackson.jaxrs.json.JacksonJsonProvider,
-                org.wso2.carbon.identity.recovery.endpoint.filter.ResponseValidationFilter,
-                org.wso2.carbon.identity.recovery.endpoint.Exceptions.RecoveryEndpointExceptionMapper
+                org.wso2.carbon.identity.api.dispatcher.core.JsonProcessingExceptionMapper,
+                org.wso2.carbon.identity.api.dispatcher.core.APIErrorExceptionMapper,
+                org.wso2.carbon.identity.api.dispatcher.core.InputValidationExceptionMapper,
+                org.wso2.carbon.identity.api.dispatcher.core.DefaultExceptionMapper,
+                org.apache.cxf.jaxrs.ext.search.SearchContextProvider
             </param-value>
         </init-param>
     </servlet>
     <servlet-mapping>
-        <servlet-name>IdentityRecoveryV0_9ApiServlet</servlet-name>
-        <url-pattern>/identity/recovery/v0.9/*</url-pattern>
+        <servlet-name>OrganizationServerV2Servlet</servlet-name>
+        <url-pattern>/o/server/v2/*</url-pattern>
     </servlet-mapping>
 
-    <!--  Servlet instance for /identity/consent-mgt/v1.0 -->
+    <!--  Servlet instance for /users/v1/me/organizations -->
     <servlet>
-        <servlet-name>IdentityConsentMgtV1ApiServlet</servlet-name>
+        <servlet-name>OrgUserServlet</servlet-name>
```

Comment:
is this required when `UsersV1Servlet` can handle these serviceClasses, as this url-pattern mapping is a subset of `/users/v1/*`